controller/integration/test/multifinalizer/multi_finalizer_test.go

```diff
@@ -66,7 +73,33 @@ func Test_MultiFinalizer(t *testing.T) {
 		}
 	}
 
-	var harnessA, harnessB *configmap.Wrapper
+	// resourceC errors once.
+	var resourceC *testresource.Resource
+	{
+		var once sync.Once
```

Comment:
I added a resourceC that fails only once. To check if `DeleteCount() == 2` for that resource.